setup.py

```diff
@@ -91,6 +91,9 @@
 
 except Exception as e:
     log.warning("Failed to get options via gdal-config: %s", str(e))
+    if os.name == "nt":
```

Comment:
@groutr consider reordering & rephrasing this a little bit - I don't know that it is helpful for us to reference a config file that isn't applicable to Windows here.

```
if os.name == "nt":
    log.info("Building on Windows requires extra arguments to setup.py to locate necessary GDAL files.\n"
             "More information available in the README.")
else:
    log.warning("Failed to get options via gdal-config: %s", str(e))
```